.github/workflows/push_checking.yml

```
@@ -18,18 +18,11 @@ jobs:
           node-version: ${{ matrix.node-version }}
 
       - name: Install dependencies
-
-        run: yarn install --frozen-lockfile
+        run: npm install
```

```suggestion
        run: npm ci
```

The same applies for running the tests. Once a package has been modified in the package-lock and committed, the rest of tasks should use exclusively that version.